src/main/java/cat/udl/eps/entsoftarch/webingogeiapi/domain/Game.java

```diff
@@ -29,6 +30,12 @@
     private boolean line;
 
     @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
-    private ZonedDateTime finishedAt, startAt;
+    private ZonedDateTime StartAt, finishedAt ;
+
+    @ManyToOne
+    @JsonIdentityReference(alwaysAsId = true)
+    private User gameRegister;
```

Comment:
The `gameRegister` should be an `Admin`. Just admins can create games as constrained in WebSecurityConfig.
The name for the relationships is quite confusing, maybe it is better to call it simply `creator`